lib/ansible/template/__init__.py

```diff
@@ -989,15 +977,14 @@ def _lookup(self, name, *args, **kwargs):
         except (AnsibleUndefinedVariable, UndefinedError) as e:
             raise AnsibleUndefinedVariable(e)
         except Exception as e:
-            if self._fail_on_lookup_errors:
-                msg = u"An unhandled exception occurred while running the lookup plugin '%s'. Error was a %s, original message: %s" % \
-                      (name, type(e), to_text(e))
-                if errors == 'warn':
-                    display.warning(msg)
-                elif errors == 'ignore':
-                    display.display(msg, log_only=True)
-                else:
-                    raise AnsibleError(to_native(msg))
+            msg = u"An unhandled exception occurred while running the lookup plugin '%s'. Error was a %s, original message: %s" % \
+                  (name, type(e), to_text(e))
+            if errors == 'warn':
+                display.warning(msg)
+            elif errors == 'ignore':
+                display.display(msg, log_only=True)
+            else:
+                raise AnsibleError(to_native(msg))
```

Comment:
since we are here, add orig_exc=e

Reply:
Wouldn't that be redundant since we include `e` in `msg` above?

Reply:
no, that just gives the message, orig_exc allows the handlers above to show the full exception when -vvv

Reply:
I still don't see how is that achieved, only usage of `orig_exc` I see is appending its stringified value to the message at
https://github.com/ansible/ansible/blob/212837defcb917e2b5613be0966a5374b3c27536/lib/ansible/errors/__init__.py#L75-L76 which is what already happened at this point for `msg` so we would just repeat the information.

Can you point to where "orig_exc allows the handlers above to show the traceback when -vvv" happens?

Reply:
on exit and -vvv the cli will check for orig_exc and print traceback

Reply:
Done. Had to change the code a bit to avoid duplicating the original error in the `AnsibleError` message.

Reply:
So this was resolved in https://github.com/ansible/ansible/pull/74108.